CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ These are garage-specific rules which are not part of the aforementioned style g
 * Python package imports should be sorted alphabetically within their PEP8 groupings. The sorting is alphabetical from left to right, ignoring case and Python keywords (i.e. `import`, `from`, `as`). Notable exceptions apply in `__init__.py` files, where sometimes this rule will trigger a circular import.
 * We prefer double-quoted strings (`"foo"`) over single-quoted strings (`'foo'`), unless there is a compelling escape or formatting reason for using single quotes
 * Add convenience imports in `__init__.py` of a package for shallow first-level repetitive imports, but not for subpackages, even if that subpackage is defined in a single `.py` file. For instance, if an import line reads `from garage.foo.bar import Bar` then you should add `from garage.foo.bar import Bar` to `garage/foo/__init__.py` so that users may instead write `from garage.foo import Bar`. However, if an import line reads `from garage.foo.bar.stuff import Baz`, *do not* add `from garage.foo.bar.stuff import Baz` to `garage/foo/__init__.py`, because that obscures the `stuff` subpackage.
-* Newly created python files should follow [PEP257](https://www.python.org/dev/peps/pep-0257/) docstring conventions and [PEP287](https://www.python.org/dev/peps/pep-0287/) docstring formatting. A good PEP257 example can be found [here](https://blog.dolphm.com/pep257-good-python-docstrings-by-example/).
+* Newly created python files should follow [PEP257](https://www.python.org/dev/peps/pep-0257/) docstring conventions and [PEP287](https://www.python.org/dev/peps/pep-0287/) docstring formatting. A good PEP257 example can be found [here](https://blog.dolphm.com/pep257-good-python-docstrings-by-example/). However, docstring for `__init__` should be included in the class docstring as suggested in the [google example](http://google.github.io/styleguide/pyguide.html).
```

Comment:
* Please link to the sphinx docs instead: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
* Google should be capitalized.